manifests/p/PsiphonInc/Psiphon/3.0/PsiphonInc.Psiphon.installer.yaml

```
@@ -0,0 +1,12 @@
+# Created with YamlCreate.ps1 v2.2.2 $debug=NVS1.CRLF.7-3-1.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: PsiphonInc.Psiphon
+PackageVersion: "3.0"
+InstallerType: portable
+Installers:
+- Architecture: x64
+  InstallerUrl: https://psiphon.ca/psiphon3.exe
+  InstallerSha256: CBBFD5C2D84ED2EC2DBEDE406D35969863871DFF3AFF7DDB1904ADEE4D2E82EA
```

```suggestion
  InstallerSha256: E98FB973657ED41EA2834B76F07DD6F65F5E1E3D8A444E655C5BDF06531F5C34
```

This seems to be updating quite fast. Looking at the pipeline's logs, around 18 hours ago it was `FAADFFF09D9C94C8B8DB911CC592A3561CBEC27D83B8878A2E087FFD930F742C`.